src/Form.svelte

```diff
@@ -20,17 +20,17 @@
 
   const validationExamples: Record<string, string> = {
     URL: "https://acme.xyz/",
-    URN: "eip155:1:0xd1bb21bd5a432d2919c82bcefe1bc7f8cc9207d9",
+    ID: "eip155:1:0xd1bb21bd5a432d2919c82bcefe1bc7f8cc9207d9",
     handle: "acme",
     id: "hydkkcf6k9be5fuszdhpqbctu3q3fuwagj874wx2puia8ti8coygh1",
-    identity: "rad:git:hnrkqdpm9ub19oc8dccx44echy75hzfsezyio",
+    identity: "rad:hnrkqdpm9ub19oc8dccx44echy75hzfsezyio",
```

Comment:
is this `ID` correct?

Reply:
Nope, we switched to multibase+base58, so all IDs start with `z`.

Reply:
so `[a-z]` -> `z[a-z]`?

Reply:
Yeah I mean you could do `z[a-zA-Z0-9]+` for a loose regex. It will include caps.